src/helpers/graphqlUtils.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+export const fromGlobalId = (globalId: string | null) => {
+  if (!globalId) return { type: '', id: '' };
+  const attrs = new Buffer(globalId, 'base64').toString().split(':');
```

Comment:
pls no base64 decoding/encoding of GraphQL IDs, they are opaque by the specification, just track GraphQL ids

Reply:
Considering the consumers of this data(data analysts) probably do not use GraphQL since it's a specification we use exclusively between our app and our graphql proxy. Isn't it more useful for them to have the articleID/categoryId that maps to the actual KB backend record directly? That can make their filtering and data browsing more comfortable. 
Granted they have the article title in the tracking payload, but title names are always more ambiguous.